packages/waveform-progress/demo/index.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
```

Comment:
Question: are we still doing storybook?

Reply:
Yeah we're still using Storybook. This is the demo app that you see while you're developing, running `npm start`